src/controller/org.controller/org.controller.js

```diff
@@ -69,6 +69,53 @@ async function getCna (req, res, next) {
   }
 }
 
+async function getUsers (req, res, next) {
+  try {
+    const shortName = req.ctx.org
+    const cnaShortName = req.ctx.params.shortname
+    const orgRepo = req.ctx.repositories.getOrgRepository()
+    const userRepo = req.ctx.repositories.getUserRepository()
+    const orgUUID = await orgRepo.getOrgUUID(cnaShortName)
+    let returned = false
+
+    if (!orgUUID) {
+      returned = true
+      logger.info(cnaShortName + ' CNA does not exist.')
+      return res.status(404).json(error.cnaDneParam(cnaShortName))
```

Comment:
More CNA wording

Reply:
This should be addressed in a separate issue since this wording change must be done throughout the services.